### PR TITLE
Construct hyperrefs in `digraph.tex`, add a `\label` to `sylow.tex`

### PR DESCRIPTION
**Pull request by chennisden**
Addresses #142

### CODE REVIEW

File: tex/frontmatter/digraph.tex [hunk 1]
```diff
@@ -3,30 +3,31 @@
...
-\halfcourse 20,28:{Ch 60-63}{Cat Th}{}
-\halfcourse 23,0:{Ch 64-69}{Alg Top 2}{}
-\halfcourse 6,10:{Ch 70-74}{Alg Geo 1}{}
-\halfcourse 6,0:{Ch 75-80}{Alg Geo 2-3}{}
-\reqhalfcourse 5,45:{Ch 81-87}{Set Theory}{}
+\reqhalfcourse 20,45:{Ch 1,3-5}{\hyperref[part:absalg]{Abs Alg}}{}
+\reqhalfcourse 55,45:{Ch 2,6-8}{\hyperref[part:basictop]{Topology}}{}
+\halfcourse 33,45:{Ch 9-15,18}{\hyperref[part:linalg]{Lin Alg}}{}
+\halfcourse 5,35:{Ch 16}{\hyperref[part:groups]{Grp Act}}{}
+\halfcourse 5,24:{Ch 17}{\hyperref[chapter:sylow]Grp Classif}{}
```
Comment (vEnhance) [inline tex/frontmatter/digraph.tex:16]: Is this supposed to be missing the curly braces around "Grp Classif"?

File: tex/frontmatter/digraph.tex [hunk 2]
```diff
@@ -3,30 +3,31 @@
 
 \bgroup
 \renewcommand{\href}[1]{} % temp disable links
+\hypersetup{linkcolor=black} % temporarily make internal refs black, so part labels are not all maroon
```
Comment (vEnhance) [inline tex/frontmatter/digraph.tex:6]: I actually think this is not super necessary (having it colored makes it a little more obvious the labels are clickable --- although I will probably add a one-sentence paragraph to explicitly say this right before the digraph itself).

File: tex/frontmatter/digraph.tex (hunk 1 above)
Comment (chennisden) [reply tex/frontmatter/digraph.tex:16]: It shouldn't be, my bad on that.

**Comment by vEnhance** [comment]
Thanks so much for implementing this.

**Pull request merged**